.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,8 @@
   "features": {
     "ghcr.io/devcontainers/features/azure-cli:1": {},
     "ghcr.io/devcontainers/features/github-cli:1": {},
-    "ghcr.io/devcontainers/features/sshd:1": {}
+    "ghcr.io/devcontainers/features/sshd:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {}
```

Comment:
This is to enable the use of the Cosmos emulator docker container, for testing. We generally test against the Emulator in a lot of our SDKs, including in CI builds: https://github.com/Azure/azure-sdk-for-go/blob/f647b1a888568b65e9bac0fcb2f1f494a660f6c1/sdk/data/azcosmos/ci.yml#L45

Reply:
Nit: I prefer alphabetical order so it's easy to find things as the list expands. And that would only be a 1-line addition as opposed to a 2-line diff.

Reply:
Agreed! I wish it were possible to make alphabetizing JSON an "auto-format on save" option :). Maybe it is...? 🤔